tenderly/middleware/throttle.ts

```diff
@@ -0,0 +1,42 @@
+import type { RequestHandler } from 'express'
+import type { ParamsDictionary } from 'express-serve-static-core'
+import type { ParsedQs } from 'qs'
+
+export const createThrottleMiddleware = <
+  P = ParamsDictionary,
+  ResBody = any,
+  ReqBody = any,
```

Comment:
I suppose we can't type these / won't realistically matter?

Reply:
yeah these typings were redundant anyway so I've removed them